lib/Conch/Controller/Login.pm

```diff
@@ -65,7 +65,7 @@ sub _create_jwt ($c, $user_id, $expires_delta = undef) {
     );
 
     # this should be returned in the json payload under the 'jwt_token' key.
-    return "$header.$payload";
+    return $header.'.'.$payload;
```

Comment:
And things like this (and the one on line 132 below) also seem to go out of their way a bit.